kaldb/src/main/java/com/slack/kaldb/chunk/ChunkManager.java

```diff
@@ -301,11 +298,12 @@ public void rollOverActiveChunk() {
                     CompletableFuture.supplyAsync(() -> chunk.query(query), queryExecutorService)
                         // TODO: this will not cancel lucene query. Use ExitableDirectoryReader in
                         // the future and pass this timeout
-                        .completeOnTimeout(errorResult, QUERY_TIMEOUT_SECONDS, TimeUnit.SECONDS))
+                        .orTimeout(QUERY_TIMEOUT_SECONDS, TimeUnit.SECONDS))
```

Comment:
let this throw a timeout exception since we catch it in exceptionally - and now we will log it as well